components/field/Field.tsx

```diff
@@ -1,10 +1,31 @@
 import styles from "./Field.module.css";
+import React from "react";
+import { APIWaste } from "../../utils/api";
 
-const Field = () => {
+function Field({ id, type, imageSrc, score }: APIWaste) {
+  const dragStart = (e) => {
+    const target = e.target;
+    e.dataTransfer.setData("fieldId", id);
+    e.dataTransfer.setData("fieldType", type);
+    e.dataTransfer.setData("fieldScore", score);
```

Comment:
If you don't want to set three different values, your could JSON.stringify it:
`event.dataTransfer.setData("waste", JSON.stringify({ id, type, score});
(you have to JSON.parse it at the other end.